jcl/src/java.base/share/classes/java/lang/String.java

```diff
@@ -8325,13 +8325,164 @@ public String indent(int n) {
 /*[ENDIF] Java12 */
 
 /*[IF Java13]*/
-	/*
+	/**
 	 * Determine if current String object is LATIN1.
 	 *
 	 * @return true if it is LATIN1, otherwise false.
 	 */
 	boolean isLatin1() {
 		return LATIN1 == coder();
 	}
+
+	/**
+	 * Format the string using this string as the format with supplied args.
+	 *
+	 * @param args
+	 *          the format arguments to use
+	 *
+	 * @return the formatted result
+	 *
+	 * @see #format(String, Object...)
+	 */
+	@Deprecated(forRemoval=true, since="13")
+	public String formatted(Object... args) {
+		return String.format(this, args);
+	}
+
+	/**
+	 * Removes the minimum indentation from the beginning of each line and
+	 * removes the trailing spaces in every line from the string
+	 *
+	 * @return this string with incidental whitespaces removed from every line
+	 */
+	@Deprecated(forRemoval=true, since="13")
+	public String stripIndent() {
+		if (isEmpty()) {
+			return "";
+		}
+		Iterator<String> iter = lines().iterator();
+		int min = Integer.MAX_VALUE;
+
+		while (iter.hasNext()) {
+			String line = iter.next();
+
+			if (!line.isBlank() || !iter.hasNext()) {
+				int count = 0;
+				int limit = Math.min(min, line.length());
+				while ((count < limit) && Character.isWhitespace(line.charAt(count))) {
+					count++;
+				}
+
+				if (min > count) {
+					min = count;
+				}
+			}
+		}
+
+		StringBuilder builder = new StringBuilder();
+		iter = lines().iterator();
+
+		while (iter.hasNext()) {
+			String line = iter.next();
+
+			if (line.isBlank()) {
+				builder.append("");
+			} else {
+				line = line.substring(min);
+				builder.append(line.stripTrailing());
+			}
+			builder.append("\n");
+		}
+
+		if (charAt(length() - 1) != '\n' && charAt(length() - 1) != '\r') {
+			builder.deleteCharAt(builder.length() - 1);
+		}
+
+		return builder.toString();
+	}
+
+	/**
+	 * Translate the escape sequences in this string as if in a string literal
+	 *
+	 * @return result string after translation
+	 *
+	 * @throws IllegalArgumentException
+	 *          If invalid escape sequence is detected
+	 */
+	@Deprecated(forRemoval=true, since="13")
+	public String translateEscapes() {
+		StringBuilder builder = new StringBuilder();
+		char[] charArray = toCharArray();
+		int index = 0;
+		int strLength = length();
+		while (index < strLength) {
+			if (charArray[index] == '\\') {
+				index++;
+				if (index >= strLength) {
+					/*[MSG "K0D00", "Invalid escape sequence detected: {0}"]*/
+					throw new IllegalArgumentException(com.ibm.oti.util.Msg.getString("K0D00", "\\")); //$NON-NLS-1$ //$NON-NLS-2$
+				}
+				int octal = 0;
+				switch (charArray[index]) {
+					case 'b':
```

Comment:
Please don't indent the body of `switch` statements (to match (most) other code).